nixos/doc/manual/development/writing-documentation.chapter.md

```diff
@@ -50,7 +50,7 @@ pandoc -f markdown_github -t docbook5 docs.md -o my-section.md
 Pandoc can also quickly convert a single `section.xml` to HTML, which is
 helpful when drafting.
 
-Sometimes writing valid DocBook is simply too difficult. In this case,
+Sometimes writing valid DocBook is too difficult. In this case,
```

Comment:
I don't see how this cold be interpreted as condescending, it's the opposite, actually.

Reply:
I never wanted to suggest it's *always* condescending :) - doesn't seem to add much, either, though.

Reply:
Yeah, most of the time is *simply* redundand.

Reply:
@rnhmjoj If you really want to keep it, you could ~~simply~~ just replace `simply` with `just` !

Reply:
No, I just mean, for something like "a monad is simply a particular monoid in the cathegory of endomorphisms" I see the value in removing simply or rewording, but for most of these I wouldn't have bothered.